projects/packages/connection/tests/php/test-class-webhooks.php

```diff
@@ -153,8 +153,6 @@ public function test_controller() {
 
 		// `handle_authorize_redirect` gets called.
 		$webhooks->controller();
-
-		static::assertNull( $controller_skipped );
```

Comment:
Why did we remove this test?

Reply:
Phan pointed out that `$webhooks->controller()` never returns a value, it's effectively `@return void`, so assigning its return value to a variable on line 133 above isn't really valid code.

Since we have no value to assign to the variable, there's no point in testing it for null here either.

Reply:
One could argue the test is ensuring that it never returns something, but I guess a proper `return` doc would flag that too.